src/sampler.jl

```diff
@@ -175,18 +180,18 @@ function sample(
     pm = progress ? ProgressMeter.Progress(n_samples, desc="Sampling", barlen=31) : nothing
     time = @elapsed for i = 1:n_samples
         # Make a step
-        t = step(rng, h, τ, t.z)
-        # Adapt h and τ; what mutable is the adaptor
+        t = transition(rng, h, κ, t.z)
```

Comment:
Any reason for changing function name `step` into `transition`? Maybe consider making it more aligned with `AbstractMCMC` function names.

Reply:
The previous `step` function in Line 41 was just a wrapper over `transition` (I pinged you there).
With the new abstraction this wrapper is unnecessary.